experimental/ast/path.go

```diff
@@ -29,6 +29,13 @@ import (
 //
 // This includes single identifiers like foo, references like foo.bar,
 // and fully-qualified names like .foo.bar.
+//
+// # Grammar
+//
+//	Path      := `.`? component (sep component)*
+//
+//	component := Ident | `(` Path `)`
+//	sep       := `.` | `/`
```

Comment:
Let's make the case consistently PascalCase here, too. Though we could use snake_case for "ident" since it's a lexical element that isn't defined in here.
```suggestion
//	Path      := `.`? Component (Sep Component)*
//
//	Component := ident | `(` Path `)`
//	Sep       := `.` | `/`
```